ReactWindows/ReactNative/Views/Web/ReactWebViewManager.cs

```diff
@@ -0,0 +1,306 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using ReactNative.Views.Web.Events;
+using System;
+using System.Collections.Generic;
+using Windows.ApplicationModel.Core;
+using Windows.UI.Core;
+using Windows.UI.Xaml.Controls;
+using Windows.Web.Http;
+
+namespace ReactNative.Views.Web
+{
+    /// <summary>
+    /// A view manager responsible for rendering webview.
+    /// </summary>
+    public class ReactWebViewManager : SimpleViewManager<WebView>
+    {
+        private const string BLANK_URL = "about:blank";
+
+        private const int CommandGoBack = 1;
+        private const int CommandGoForward = 2;
+        private const int CommandReload = 3;
+
+        private readonly Dictionary<int, string> _injectedJS = new Dictionary<int, string>();
+
+        /// <summary>
+        /// The name of the view manager.
+        /// </summary>
+        public override string Name
+        {
+            get
+            {
+                return "RCTWebView";
+            }
+        }
+
+        /// <summary>
+        /// The commands map for the webview manager.
+        /// </summary>
+        public override IReadOnlyDictionary<string, object> CommandsMap
+        {
+            get
+            {
+                return new Dictionary<string, object>
+                {
+                    { "goBack", CommandGoBack },
+                    { "goForward", CommandGoForward },
+                    { "reload", CommandReload },
+                };
+            }
+        }
+
+        /// <summary>
+        /// Sets whether JavaScript is enabled or not.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="enabled"></param>
```

Comment:
I meant for the empty param comment.

---
In reply to: [60739117](https://github.com/ReactWindows/react-native/pull/372#discussion_r60739117) [](ancestors = 60739117)